server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -4,79 +4,13 @@ import           Data.Aeson
 import           Data.Has
 import           Hasura.Prelude
 
-import qualified Data.HashMap.Strict                 as Map
-import qualified Data.HashSet                        as Set
-import qualified Data.Text                           as T
-import qualified Language.GraphQL.Draft.Syntax       as G
+import qualified Data.HashMap.Strict           as Map
+import qualified Data.HashSet                  as Set
+import qualified Language.GraphQL.Draft.Syntax as G
 
-import           Hasura.GraphQL.Resolve.ContextTypes
+import           Hasura.GraphQL.Resolve.Types
 import           Hasura.GraphQL.Validate.Types
-import           Hasura.RQL.Types.BoolExp
-import           Hasura.RQL.Types.Common
 import           Hasura.RQL.Types.Permission
-import           Hasura.SQL.Types
-
-type OpCtxMap = Map.HashMap G.Name OpCtx
-
-data InsOpCtx
```

Comment:
These types are related to the `Resolve` module and they have been moved to `Types.hs` in `Resolve`.

Reply:
At the same time `ContextTypes.hs` has been renamed/merged into `Types.hs` with the above inclusions.